packages/realm/bindgen/src/templates/node-wrapper.ts

```diff
@@ -20,14 +20,15 @@ import { bindModel, Property } from "@realm/bindgen/bound-model";
 import { TemplateContext } from "@realm/bindgen/context";
 
 import { doJsPasses } from "../js-passes";
+import { eslintFormatter } from "../formatters/eslint-formatter";
```

Comment:
I'm not sure it is worth introducing a new directory for just this one file. Personally, since the formatters are tiny, I would just put all of them in a single `../formaters.ts` file, even if we had more than one.

But at the very least, lets remove some redundancy in `src/formatters/eslint-formatter.ts`. If you want to keep one file per formatter, I'd go either `src/formatters/eslint.ts` or just `src/eslint-formatter.ts`. Slight preference for the latter if you don't want just `src/formatters.ts`

Reply:
Moved it one level up again 👍